pkg/oc/bootstrap/docker/up.go

```diff
@@ -501,40 +501,41 @@ func (c *ClusterUpConfig) Start(out io.Writer) error {
 	}
 	taskPrinter.Success()
 
+	installContext, err := componentinstall.NewComponentInstallContext(c.openshiftImage(), c.BaseDir)
+	if err != nil {
+		return err
+	}
+
 	// TODO, now we build up a set of things to install here.  We build the list so that we can install everything in
 	// TODO parallel to avoid anyone accidentally introducing dependencies.  We'll start with migrating what we have
 	// TODO and then we'll try to clean it up.
 	registryInstall := &registry.RegistryComponentOptions{
-		OCImage:         c.openshiftImage(),
```

Comment:
The question I left Michal on IRC is that: shouldn't we include `ServerLogLevel` and `ImageFormat` to `InstallContext` per this comment from David https://github.com/openshift/origin/pull/19215#discussion_r179311599. I'm hoping these two values will be needed by every component, even though we're not setting them currently.

Reply:
added them